content/2021/2020-01-21_whats-new/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "What has 2020 brought to us?"
+description = "A lot has been done, a lot more to do!"
+
+[taxonomies]
+tags = ["announcement", "changelog", "improvements", "runner", "new-language", "documentation", "moderation"]
++++
+
+
+Year 2020 and beginning of 2021 was a very busy period for Codewars maintainers. We worked hard on improving the platform, and on fixing existing bugs which were getting in a way of further improvements. We were looking for ways and ideas how to get closer to Codewars users. At the same time we managed to introduce a few new languages to train on, upgraded some language versions, and provided some means to support users and new-comers.
+
+
+## New blog
+
+We migrated the Codewars blog to new platform, and plan to use it for announcements for the Codewars community. We will post here information and updates about new features, improvements, and we will call for your help when we need one. Blog posts will show up on the dashboard of the main site, what will hopefully help to bring the Codewars closer to its users.
+
+
+## New languages
+
+2020 brought a large set of new languages which users can train on, create new kata, and translate existing kata to! Most of them is still in beta stage, and contibutions in form of new kata or translations will be very helpful. Many languages were also updated to new versions. Some of the added languages are Lean, COBOL, Perl, Pascal, but these are not the only ones! More details will be provided in a dedicated post.
+
+Many thanks to all users who provided their support on setting up the code runner and created new content for introduced languages!
+
+
+## Refactoring and improvements
+
+Refactoring was probably the biggest topic of all activities related to Codewars conducted in recent time. Thanks to code cleanup, reducing technical debt, and even total rewrite of some elements, it was possible to improve performance in the backend, and improve user experience and navigation on the frontend. Getting rid of some messy code also enabled maintainers to provide futher improvements in the future, which, until now, were very difficult to add.
+
+
+## Documentation
+
+Addressing the needs of new users and questions of many experienced ones, new [documentation site](https://docs.codewars.com/) has been launched. Documentation is meant to be created by community, so everyone can write articles, review drafts, suggest improvements, or fix any issues using [codewars/docs](https://github.com/codewars/docs) GitHub repository. Documentation is still a work in progress, but when it gets into an acceptably good shape, it will replace the old Codewars wiki.
```

Comment:
```suggestion
Addressing the needs of new users and questions of many experienced ones, a new [documentation site](https://docs.codewars.com/) has been launched. Documentation is meant to be created by the community, so everyone can write articles, review drafts, suggest improvements, or fix any issues using [codewars/docs](https://github.com/codewars/docs) GitHub repository. The documentation is still a work in progress, but when it gets into an acceptably good shape, it will replace the old Codewars wiki.
```